.github/CODEOWNERS

```diff
@@ -10,3 +10,4 @@
 
 # Misc
 /generators/aws-sdk-rbs-generator @ksss
+/generators/rails-generator @ksss
```

Comment:
How about assigning you (and @pocke) to `/gems/actionpack` and so on too? A generator change affects under the `gems` directory, so I think you should have permission to them also.